codalab/worker/bundle_manager.py

```diff
@@ -441,4 +441,10 @@ def parse_and_min(to_value, request_string, default_value, max_value):
         resources['request_network'] = (bundle.metadata.request_network or
                                         self._default_request_network)
 
+        resources['request_cpus'] = (bundle.metadata.request_cpus or
+                                        self._default_request_cpus)
+
+        resources['request_gpus'] = (bundle.metadata.request_gpus or
+                                     self._default_request_gpus)
+
```

Comment:
Are you referring to just the CPU and GPU flags or just this set of code in general?